WordPress/Classes/ViewRelated/Post/PostSettingsViewController.m

```diff
@@ -764,6 +777,16 @@ - (UITableViewCell *)configureGeolocationCellForIndexPath:(NSIndexPath *)indexPa
     return cell;
 }
 
+- (UITableViewCell *)configureExcerptCellForIndexPath:(NSIndexPath *)indexPath
+{
+    UITableViewCell *cell = [self getWPTableViewCell];
```

Comment:
+ the Swift convention for factory method naming (which may cover this?) is `makeX`.